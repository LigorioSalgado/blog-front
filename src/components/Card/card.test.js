import React from 'react';
import {shallow} from "enzyme";

import {Card} from '../Card';



describe("<Card/>", () => {

    it("Debe ejecutar render correctamente",() => {
        const component =  shallow(<Card title="prueba"/>)
        expect(component).toMatchSnapshot();
    })

    it("Debe pintar props",() => {
        const post = {
            
            title:"test",
            image:"image test"
        }

         const component =  shallow(<Card 
                title={post.title}
                image={post.image}
             />)
        
        expect(component.find(".card-title").text()).toBe(post.title);
   })

})
