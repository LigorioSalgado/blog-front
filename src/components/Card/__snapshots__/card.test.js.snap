// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card/> Debe ejecutar render correctamente 1`] = `ShallowWrapper {}`;
